istioctl/cmd/analyze.go

```diff
@@ -365,7 +366,12 @@ func gatherFile(f string) (local.ReaderSource, error) {
 	if err != nil {
 		return local.ReaderSource{}, err
 	}
-	runtime.SetFinalizer(r, func(x *os.File) { x.Close() })
+	runtime.SetFinalizer(r, func(x *os.File) {
+		err = x.Close()
+		if err != nil {
+			log.Infof("file : %s is not closed successfully with error: %v", f, err)
```

Comment:
Consider replacing "log.Infof" with "log.Errorf" here and other newly added log.Infof for errors.

Reply:
We don't want to add an error message for this.